IsraelHiking.Web/sources/application/components/sidebar/sidebar.css

```diff
@@ -9,26 +9,41 @@
     overflow-x: hidden;
     color: black;
     direction: ltr;
+    padding-left: 55px;
+    padding-right: 20px;
     background: #dddddd !important;
-    width: 0;
     z-index: 1010;
     -webkit-transition: all 0.5s ease;
     -moz-transition: all 0.5s ease;
     -o-transition: all 0.5s ease;
     transition: all 0.5s ease;
+    transform: translateX(-100%);
 }
 
+    #sidebar-wrapper:not(.toggled) {
+        width: 295px;
+    }
+
     #sidebar-wrapper.toggled {
-        width: 280px;
-        padding-left: 45px;
+        transform: translateX(0);
+    }
+
```

Comment:
Can we make row-hover definition in one place or we have to do it in multiple places?
Meaning, can we move the row-hover from layerssidebar.css here so it will affect all rows?